src/plugins/walletConnectToDapps/components/header/DappHeaderMenuSummaryV2.tsx

```diff
@@ -45,26 +45,19 @@ export const DappHeaderMenuSummaryV2 = () => {
     if (!session || !web3wallet || !core) return
 
     /*
-     TODO: this is a hack to clear out all sessions, as the MVP supports only one at a time.
+     TODO: we only support one session at a time (the most recent)
      In the future we want to support multiple pairings and sessions at once.
      */
     const activeTopics = Object.values(web3wallet.getActiveSessions()).map(session => session.topic)
-    for await (const topic of activeTopics) {
-      try {
-        // This currently throws, and does not work: https://github.com/WalletConnect/walletconnect-monorepo/issues/1772
-        await web3wallet.disconnectSession({ topic, reason: getSdkError('USER_DISCONNECTED') })
-      } catch (e) {
-        throw new Error(`Error disconnecting session: ${e}, topic: ${topic}`)
-      }
-    }
+    const mostRecentTopic = activeTopics.slice(-1)[0]
```

Comment:
🧠

Reply:
Prefer `activeTopics.length -1` as an indexor to access the last item, mostly because `slice` and `splice` are so similar and it's easy to use the latter, but obviously fine

Reply:
@0xdef1cafe's little concision trick.

Reply:
ya either or